Sources/SwiftUINavigation/Internal/Deprecations.swift

```diff
@@ -2,6 +2,39 @@
   import SwiftUI
   @_spi(RuntimeWarn) import SwiftUINavigationCore
 
+  // NB: Deprecated after 1.2.1
+
+  @available(iOS 15, macOS 12, tvOS 15, watchOS 8, *)
+  extension View {
+    @available(*, deprecated, renamed: "alert(_:title:actions:message:)")
```

Comment:
Should this include `item`?

```suggestion
    @available(*, deprecated, renamed: "alert(item:title:actions:message:)")
```